--- config.def.h.orig	2017-01-21 00:03:50.799369850 +0100
+++ config.def.h	2017-01-21 00:06:24.612710136 +0100
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char font[] = "Roboto Mono:style=Light:size=11:antialias=true:autohint=true";
 static int borderpx = 2;
 
 /*
@@ -185,6 +185,7 @@
 	{ MODKEY|ShiftMask,     XK_V,           clippaste,      {.i =  0} },
 	{ MODKEY,               XK_Num_Lock,    numlock,        {.i =  0} },
 	{ XK_ANY_MOD,           XK_F6,          swapcolors,     {.i =  0} },
+        { MODKEY,               'u',            externalpipe,   {.v = "xurls | dmenu -l 10 | xargs -r open" } },
 };
 
 /*
